src/components/ha-camera-stream.ts

```diff
@@ -163,6 +184,26 @@ class HaCameraStream extends LitElement {
     }
   }
 
+  private async _renderHLSExoPlayer(url: string) {
+    this._resizeExoPlayerListener = () => this._resizeExoPlayer();
+    window.addEventListener('resize', this._resizeExoPlayerListener);
+    // https://github.com/typescript-eslint/typescript-eslint/issues/1642
+    // eslint-disable-next-line
+    setTimeout(this._resizeExoPlayerListener,200);
+    this._videoEl.style.visibility = "hidden";
+    this.hass!.auth.external!.fireMessage({
+      type: "play_hls",
```

Comment:
So we're adding 2 new messages to the external bus: `play_hls` and `stop_hls`. 

We're adding 1 new property to the external config which is `hasExoPlayer`.

We should make sure that the commands are matching the `hasX`, and let's follow the pattern of [`config_screen/show`](https://developers.home-assistant.io/docs/frontend/external-bus#show-config-screen-config_screenshow) and call it `exoplayer/play` and `exoplayer/stop`. 

We should also have the `exoplayer/play` command return an object to indicate if the starting of exoplayer was successful, so we can do some error handling in the UI.

Reply:
Oh and `exoplayer/resize`

Reply:
Sure. Do we need to do the error handling now or can I just call "await sendMessage()" for now?
For the naming, should we call it exoplayer/play or exoplayer/play_hls? On the Android side we are specifically waiting for an HLS uri. If by chance some other media player card wants to play some other kind of media through exoplayer, it would have to pass a different message.